### DISCUSSION TITLE
Understanding Image Compatibility in ComfyUI Workflows: PNG vs. JPEG Formats

### DISCUSSION THREAD

**Discussion by whitevamp** [Comfy-Org/ComfyUI#1533] [Q&A]
I am new to ComfyUI, and I have been using SD. I have a question regarding image compatibility within the workflow. I've noticed that when I drop a PNG image into the workflow, it displays correctly with the associated workflow. However, every time I attempt to drop a JPEG image into the workflow, nothing happens. I suspect that JPEG images may not be supported in this context. Is my assumption correct, or could there be something I've overlooked, perhaps related to the PNG format?

**Comment by coreyryanhanson** [comment]
What do you mean by "displays correctly with the associated workflow" specifically?

Do you mean pulls up all associated nodes that were involved in creating it?

Side note: If you are saving renders directly to jpegs, I wouldn't recommend it in general regardless of the way ComfyUI handles things. It is a little more space efficient sure, but it is a lossy format and every save degrades the image slightly. Better to work in PNG and convert to jpeg at the very end where you have to deliver something that won't take pngs.

**Comment by okolenmi** [comment]
PNG can support additional data inside of image. You can even read this data, when you open PNG file with text editor. It looks like, JPEG can't support such metadata inside of file.
